src/silverlabnwb/nwb_file.py

```diff
@@ -446,17 +452,19 @@ def read_speed_data(self, file_name):
         Pandas data table, and initial_time is the experiment start time, which sets the
         session_start_time for the NWB file.
         """
-        self.log('Loading speed data from {}', file_name)
-        assert os.path.isfile(file_name)
-        speed_data = pd.read_csv(file_name, sep='\t', header=None, usecols=[0, 1, 2, 3], index_col=0,
-                                 names=('Date', 'Time', 'Trial time', 'Speed'),
-                                 dtype={'Trial time': np.int32, 'Speed': np.float32},
-                                 parse_dates=[[0, 1]],  # Combine first two cols
-                                 dayfirst=True, infer_datetime_format=True,
-                                 memory_map=True)
-        initial_offset = pd.Timedelta(microseconds=speed_data['Trial time'][0])
-        initial_time = speed_data.index[0] - initial_offset
-        return speed_data, initial_time
+        self.log('Attempting to load speed data from {}', file_name)
+        if os.path.isfile(file_name):
+            speed_data = pd.read_csv(file_name, sep='\t', header=None, usecols=[0, 1, 2, 3], index_col=0,
+                                     names=('Date', 'Time', 'Trial time', 'Speed'),
+                                     dtype={'Trial time': np.int32, 'Speed': np.float32},
+                                     parse_dates=[[0, 1]],  # Combine first two cols
+                                     dayfirst=True, infer_datetime_format=True,
+                                     memory_map=True)
+            initial_offset = pd.Timedelta(microseconds=speed_data['Trial time'][0])
+            initial_time = speed_data.index[0] - initial_offset
+            return speed_data, initial_time
+        else:
+            return None, None  # TODO throw warning here!!
```

Comment:
I'm not sure we need to warn if the file isn't there. Except that if we have neither speed data nor a modern header we need to bail out with no start time.